docs/detections/prebuilt-rules/tune-rule-signals.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[tuning-detection-signals]]
 == Tune detection rules
 
-Using the {es-sec-app}, you can tune prebuilt and custom detection rules to optimize alert generation. To reduce noise, you can:
+Using the {security-app}, you can tune prebuilt and custom detection rules to optimize alert generation. To reduce noise, you can:
```

Comment:
Same question as above - why is {security-app} more appropriate than {elastic-sec} in this situation?